src/pages/transaction-signing/components/confirm-send-drawer.tsx

```diff
@@ -85,6 +104,98 @@ const Actions: React.FC<{ transaction: StacksTransaction | null; handleCancel: (
   );
 };
 
+interface TxSettingsProps {
+  transaction: StacksTransaction | null;
+  setTransaction: (transaction: StacksTransaction) => void;
+  amount: number;
+  memo: string;
+  recipient: string;
+}
+
+const TxSettings: React.FC<TxSettingsProps> = ({
+  transaction,
+  setTransaction,
+  amount,
+  memo,
+  recipient,
+}) => {
+  const MAX_LENGTH_BIG_NUMBER = 16;
+  const [nonceOverride, setNonceOverride] = React.useState(
+    transaction?.auth?.spendingCondition?.nonce || 0
+  );
+  const network = useAtomValue(currentStacksNetworkState);
+  const account = useAtomValue(currentAccountState);
```

Comment:
Aren't there wrapper hooks that we can use to import these values, rather than importing from the store directly?

Reply:
I will change it.